flash/core/data/data_source.py

```diff
@@ -543,3 +545,65 @@ def _get_classes(self, data):
             classes = data.distinct(label_path)
 
         return classes
+
+
+class LabelStudioDataset(Dataset):
```

Comment:
This logic should be part of the DataSource. The idea is to get rid of Dataset which are too static objects.

Reply:
Fixed